sidebars.js

```diff
@@ -11,94 +11,107 @@ const about = [
   },
 ]
 
-const docs = [
+const manual = [
   {
-    'Getting Started': [
-      'getting-started/prerequisites',
-      {
-        Setup: [
-          'getting-started/setting-up-linux',
-          'getting-started/setting-up-macos',
-          'getting-started/setting-up-windows',
-        ],
-      },
-      'getting-started/beginning-tutorial',
-    ],
-    Development: [
-      'development/security',
-      'development/development-cycle',
-      'development/updating-dependencies',
-    ],
-    Architecture: [
-      'architecture/build-tools-node',
-      'architecture/frontend-frameworks',
-      {
-        Patterns: [
-          'architecture/patterns/brownfield',
-          'architecture/patterns/isolation',
-        ],
-      },
-      {
-        Recipes: [
-          'architecture/recipes/about-recipes',
-          'architecture/recipes/hermit',
-          'architecture/recipes/bridge',
-          'architecture/recipes/cloudish',
-          'architecture/recipes/cloudbridge',
-          'architecture/recipes/lockdown',
-          'architecture/recipes/multiwin',
-          'architecture/recipes/glui',
-        ],
-      },
-    ],
-    Guides: [
-      'guides/cli',
-      'guides/command',
-      'guides/events',
-      'guides/plugin',
-      'guides/icons',
-      'guides/splashscreen',
-      'guides/window-customization',
-      'guides/menu',
-      'guides/system-tray',
-    ],
-    Debugging: ['debugging/debugging'],
-    Testing: [
-      'testing/mocking',
-      {
-        'WebDriver Testing': [
-          'testing/webdriver/introduction',
-          {
-            'Example Application': [
-              'testing/webdriver/example/setup',
-              'testing/webdriver/example/webdriverio',
-              'testing/webdriver/example/selenium',
-            ],
-          },
-          'testing/webdriver/ci',
-        ],
-      },
-    ],
-    Building: [
-      'building/introduction',
-      'building/cross-platform',
-      'building/app-size',
-      'building/sidecar',
-      'building/debian',
-    ],
-    Distribution: [
-      'distribution/publishing',
-      'distribution/windows',
-      'distribution/linux',
-      'distribution/macos',
-      'distribution/sign-macos',
-      'distribution/updater',
-      'distribution/sign-windows',
-    ],
+    type: 'autogenerated',
+    dirName: 'manual', // '.' means the current docs folder
   },
-  'faq',
 ]
 
+  // {
```

Comment:
Feel free to remove all of this if you're happy with the way the sidebar renders ;)